### PR TITLE
[webview_flutter] Fix typo

### PR DESCRIPTION
**Pull request by fzyzcjy**
Just one char fix :) Realize this when reading through the code trying to fix a real bug

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [ ] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/plugins/blob/main/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/main/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[the auto-formatter]: https://github.com/flutter/plugins/blob/main/script/tool/README.md#format-code
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Review by bparrishMines — CHANGES_REQUESTED** [review]
Nice catch, thanks! This still needs a minor version bump since it changes code the client uses. Otherwise, it looks good.

**Comment by fzyzcjy** [comment]
> This still needs a minor version bump since it changes code the client uses

Hi, it looks like it is a private method (start with underline) so client can never touch it?

**Comment by fzyzcjy** [comment]
@bparrishMines Oh yes, I did not realize it does not fall under any category (surprisingly), thanks for pointing out.

**Review by bparrishMines — APPROVED** [review]
LGTM

@stuartmorgan for secondary review

File: packages/webview_flutter/webview_flutter/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * Fixes avoid_redundant_argument_values lint warnings and minor typos.
 * Ignores unnecessary import warnings in preparation for [upcoming Flutter changes](https://github.com/flutter/flutter/pull/104231).
 * Updates references to the obsolete master branch.
+* Fix typo from lowercase to uppercase
```
Comment (stuartmorgan) [inline packages/webview_flutter/webview_flutter/CHANGELOG.md:8]: This needs to follow the changelog style guide linked from the PR description.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]
> I did not realize it does not fall under any category (surprisingly)

The reason there's no exception category for "changes to runtime code that don't actually affect anything" is that in practice—since it would be entirely subject to human judgment—it would actually be "changes to runtime code that weren't *intended* to affect anything".

I've seen (and been the cause of) many examples over the years of changes that weren't expected to change anything, but actually did. Batching those with other changes undermines the
> makes regressions easier to pinpoint

part of the policy's goals.

File: packages/webview_flutter/webview_flutter/CHANGELOG.md (hunk 1 above)
Comment (fzyzcjy) [reply packages/webview_flutter/webview_flutter/CHANGELOG.md:8]: done

**Review by stuartmorgan — APPROVED** [review]
LGTM

You'll need to merge in the latest `main` to fix the unrelated CI failures, and request a test exemption from Hixie.

**Comment by Hixie** [comment]
test-exemption: code refactor with no semantic change

**Review by Hixie — APPROVED** [review]
LGTM

**Pull request merged**